### DISCUSSION TITLE
Make NetworkTransform and NetworkAnimator share a packet and update cycle

### DISCUSSION THREAD

**Discussion by Ovahlord** [FirstGearGames/FishNet#656] [Feature Request]
So I've been doing some character movement related things recently and I stumbled across a slightly inconvenient issue:
When using the Animator and Animation Events (for example one when playing the landing animation when hitting the ground), the NetworkTransform may be still behind position-wise.

How is this a problem?
When spawning objects locally via animation event and you're in need of an up-to-date transform position, you may end up with a position that is behind, causing us having to deal with unreliable data.

Here is an example:

https://github.com/FirstGearGames/FishNet/assets/18347559/2ff1a08a-3613-4575-bfc9-248bebfba5e3

As you can see on the large screen, the other client is jumping and suposed to spawn a particle system at the foot of the characters (transform.position) when landing, since the landing fires an Animation Event also via NetworkAnimator, you can clearly see the consequences of having two network components not synchronizing at the same time: the particles spawn mid air on another player's client because the animator triggers the event already while the network transform still has work to do.

So my request would be implementing a synchronization mode that allows linking NetworkTransform and NetworkAnimator to synchronize their timings better.

**Comment by FirstGearGames** [comment]
They already do share the same packet if modified at the same time. The update cycle is also relatively the same long as the interpolation values are identical.
